### PR TITLE
[toolbars-new/editorCommands] fix: code-block action

### PR DESCRIPTION
**Pull request by tamrat-bay**

### CODE REVIEW

File: packages/editor-common/web/src/Utils/draftUtils.ts [hunk 1]
```diff
@@ -10,6 +10,8 @@ import {
   RawDraftEntity,
   EditorChangeType,
   EntityInstance,
+  genKey,
```
Comment (tamrat-bay) [inline packages/editor-common/web/src/Utils/draftUtils.ts:13]: Remove

File: packages/editor-common/web/src/Utils/draftUtils.ts [hunk 2]
```diff
@@ -859,3 +860,81 @@ export function setNativeSelectionToBlock(block: ContentBlock) {
...
+    anchorOffset: 0,
+    focusKey: lastKey,
+    focusOffset: contentState.getBlockForKey(lastKey).getLength(),
+  });
+
+  return Modifier.replaceWithFragment(contentState, target, fragment);
+}
+
+function createEmptyBlock() {
+  return ContentState.createFromText('')
```
Comment (shaulgo) [inline packages/editor-common/web/src/Utils/draftUtils.ts:901]: createEmptyBlocksInFragment(numberOfBlocks)

File: packages/editor-common/web/src/Utils/draftUtils.ts [hunk 3]
```diff
@@ -859,3 +860,81 @@ export function setNativeSelectionToBlock(block: ContentBlock) {
   selection?.removeAllRanges();
   selection?.addRange(range);
 }
+
+function insertTextToBlock(block, contentState, text) {
+  const key = block.getKey();
+  const targetLength = block.getLength();
+  const newLineTarget = SelectionState.createEmpty(key)
```
Comment (shaulgo) [inline packages/editor-common/web/src/Utils/draftUtils.ts:867]:
```suggestion
  const selection = createSelection...
```

File: packages/editor-common/web/src/Utils/draftUtils.ts [hunk 4]
```diff
@@ -859,3 +860,81 @@ export function setNativeSelectionToBlock(block: ContentBlock) {
...
+    .set('anchorOffset', targetLength)
+    .set('focusOffset', targetLength);
```
Comment (shaulgo) [inline packages/editor-common/web/src/Utils/draftUtils.ts:869]: this are 0

**Review by shaulgo — APPROVED** [review]

**Pull request merged**